src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -44,6 +43,7 @@ private class SuggestedActionsSource : ForegroundThreadAffinitizedObject, ISugge
 
             // mutable state
             private Workspace _workspace;
+            private IWorkspaceStatusService _workspaceStatusService;
```

Comment:
I believe anything we give out to an external component such as ISuggestedActionSource, we usually cut things out (set null) as soon as it gets disconnected/diposed since we had so many cases where we got leaked because other components held onto us even after it got disposed.